alica_tests/src/test/test_alica_tracing.cpp

```diff
@@ -89,7 +84,7 @@ TEST_F(AlicaTracingTest, runTracing)
     auto log = logs.front(); // MasterPlan
     EXPECT_EQ(log.first, "Plan");
     EXPECT_EQ(log.second, "true");
-    logs.pop();
+    logs.erase(logs.begin());
```

Comment:
Validating logs and tags is low priority and non-critical one can even be skipped. we should be validating parent child relationships.